### PR TITLE
Use <For/> in `RangeSlider` documentation

### LINKED ISSUE #3736: Use `<For />` in `RangeSlider` documentation
### Provide the exact quote of the error or issue
We should use `<For />` component when repeating a component
e.g.
```tsx
<For each={items}>{(item, index) => <Text key={index} fontWeight="bold">{item}</Text>}</For>
```

https://yamada-ui.com/components/forms/range-slider

### Does the docs page already exist? Please link to it.

_No response_

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### PR DESCRIPTION
**Pull request by kaehehehe**
Closes #3736

## Description

Fixed the repetition part in the `RangeSlider` documentation to use the `For` component.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: docs/contents/components/forms/range-slider/index.ja.mdx [hunk 1]
```diff
@@ -272,39 +292,19 @@ return (
...
-    <RangeSliderMark value={25} w="10" ml="-5">
-      25%
-    </RangeSliderMark>
-    <RangeSliderMark value={50} w="10" ml="-5">
-      50%
-    </RangeSliderMark>
-    <RangeSliderMark value={75} w="10" ml="-5">
-      75%
-    </RangeSliderMark>
+    <For each={["sm", "md", "lg"]}>
```
Comment (108yen) [inline docs/contents/components/forms/range-slider/index.ja.mdx:295]: image

**Review by 108yen — CHANGES_REQUESTED** [review]

File: docs/contents/components/forms/range-slider/index.ja.mdx (hunk 1 above)
Comment (kaehehehe) [reply docs/contents/components/forms/range-slider/index.ja.mdx]: Oops. Thanks for pointing that out!

File: docs/contents/components/forms/range-slider/index.ja.mdx [hunk 2]
```diff
@@ -326,15 +324,13 @@ return (
...
-    <RangeSliderMark value={25} w="10" ml="-5">
-      25%
-    </RangeSliderMark>
-    <RangeSliderMark value={50} w="10" ml="-5">
-      50%
-    </RangeSliderMark>
-    <RangeSliderMark value={75} w="10" ml="-5">
-      75%
-    </RangeSliderMark>
+    <For each={[25, 50, 75]}>
```
Comment (108yen) [inline docs/contents/components/forms/range-slider/index.ja.mdx:327]: And while you're at it, can you remove `{" "}`?

File: docs/contents/components/forms/range-slider/index.mdx [hunk 3]
```diff
@@ -328,15 +326,13 @@ return (
...
-    <RangeSliderMark value={25} w="10" ml="-5">
-      25%
-    </RangeSliderMark>
-    <RangeSliderMark value={50} w="10" ml="-5">
-      50%
-    </RangeSliderMark>
-    <RangeSliderMark value={75} w="10" ml="-5">
-      75%
-    </RangeSliderMark>
+    <For each={[25, 50, 75]}>
```
Comment (108yen) [inline docs/contents/components/forms/range-slider/index.mdx:329]: Here's another unnecessary `{" "}`.

File: docs/contents/components/forms/range-slider/index.ja.mdx (hunk 2 above)
Comment (kaehehehe) [reply docs/contents/components/forms/range-slider/index.ja.mdx:327]: @108yen 
The blank space that was originally there. I didn't add it.
スクリーンショット 2024-12-12 105545

It seems like there won't be any issues even if we remove the blank space... Should we go ahead and remove it?

Comment (108yen) [reply docs/contents/components/forms/range-slider/index.ja.mdx:327]: @kaehehehe
I think it was just a mistake, so could you please delete it?
If it doesn't work, Mr. Yamada will reject it!

Comment (kaehehehe) [reply docs/contents/components/forms/range-slider/index.ja.mdx:326]: Fixed! Please rereview.

File: docs/contents/components/forms/range-slider/index.mdx (hunk 3 above)
Comment (kaehehehe) [reply docs/contents/components/forms/range-slider/index.mdx:328]: Fixed.

**Review by 108yen — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**